renovate.json

```diff
@@ -9,7 +9,8 @@
     "org.apache.maven.resolver:maven-resolver-transport-file",
     "org.apache.maven.resolver:maven-resolver-transport-http",
     "org.apache.maven.resolver:maven-resolver-transport-wagon",
-    "org.eclipse.sw360:client"
+    "org.eclipse.sw360:client",
+    "org.springframework:spring-core"
```

Comment:
Commit message: Sounds like this would ignore only ignore Spring 6 but in fact this ignores spring completely, also Spring 5 maintenance releases. And maybe say "currently stuck".